pvlib/irradiance.py

```diff
@@ -661,9 +668,18 @@ def isotropic(surface_tilt, dhi):
         <=180. The tilt angle is defined as degrees from horizontal
         (e.g. surface facing up = 0, surface facing horizon = 90)
 
+    surface_azimuth : numeric
+        Surface azimuth angles in decimal degrees. surface_azimuth must
+        be >=0 and <=360. The azimuth convention is defined as degrees
+        east of north (e.g. North = 0, South=180 East = 90, West = 270).
+
     dhi : numeric
         Diffuse horizontal irradiance in W/m^2. DHI must be >=0.
 
+    horizon_profile : list (optional)
+        A list of (azimuth, dip_angle) tuples that defines the horizon
+        profile
+
```

Comment:
Do you  require 361 values here?

Reply:
no. The restructured code might make this easier to see.